precompiles/warden/events.go

```diff
@@ -601,7 +602,7 @@ func (p Precompile) GetNewSignRequestEvent(ctx sdk.Context, _ *common.Address, n
 
 	b.Write(evmoscmn.PackNum(reflect.ValueOf(big.NewInt(int64(typedEvent.GetKeyId())))))
 	b.Write(append(make([]byte, 12), creatorAddress.Bytes()...))
-
+	b.Write(evmoscmn.PackNum(reflect.ValueOf(big.NewInt(int64(typedEvent.GetBroadcastType())))))
```

Comment:
To add on this: I saw we usually convert numbers into int64, then wrap them .Int. Is it needed? Can’t we just pass smaller ints?

Reply:
will check once I find the root cause of why integration test (WardenPrecompileAction) is failing

Reply:
Yeah, it can be done without wrapping, integration test WardenPrecompileAction passes

I removed redundant mapping in this diff (precompiles/warden/events.go), but I think we should leave similar mapping in precompiles/warden/tx_types.go (newMsgNewSignRequest function), as it fullfills kind of validation purpose. Is this okay?

Reply:
@artur-abliazimov